ggshield/core/config.py

```diff
@@ -258,7 +258,8 @@ def to_dict(self) -> Dict:
 
 @dataclass
 class InstanceConfig:
-    account: AccountConfig  # Only handle 1 account per instance for the time being
+    # Only handle 1 account per instance for the time being
+    account: Optional[AccountConfig]
```

Comment:
Why do we need `account` to become optional?

Reply:
in `login.py` Julien instanciates the config with no account with this comment
> account is initialized as None because the instance must exist in
> the config before using the client

Reply:
Ah right, I think I can fix this in my refacto PR by using the lower-level [`create_client()`](https://github.com/GitGuardian/ggshield/blob/agateau/config-fixes/ggshield/core/client.py#L29) function. Going to give it a try.